src/vehicle_display.cpp

```diff
@@ -499,8 +499,8 @@ void vehicle::print_speed_gauge( const catacurses::window &win, const point &p,
                    static_cast<int>( std::log10( static_cast<double>( std::abs( t_speed ) ) ) ) + 2 );
 
     const std::string type = get_option<std::string> ( "USE_METRIC_SPEEDS" );
-    mvwprintz( win, p, c_light_gray, "%s :", type );
+    mvwprintz( win, p, c_light_gray, pgettext( "speed gauge unit", "%s :" ), type );
```

Comment:
`pgettext( "speed gauge unit", "%s :" )` is not translating speed unit, it is translating the colon character.